src/docfx/build/toc/BuildTableOfContents.cs

```diff
@@ -42,12 +42,9 @@ public static void Build(Context context, Document file)
             {
                 if (context.Config.OutputType == OutputType.Html)
                 {
-                    if (file.IsHtml)
```

Comment:
TOC is a component in docs-ui template but is content in pdf template, this if check is still needed then?

Reply:
TOC can always be able to generated to a HTML like `conceptual`, `landingData`, but whether to generate a HTML should be decided by the outputType